config/300-channel-eventing.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: channels.eventing.knative.dev
+  labels:
+    eventing.knative.dev/release: devel
+    knative.dev/crd-install: "true"
+spec:
```

Comment:
As per the earlier design, shouldn't this be living in the messaging.knative.dev space?
https://github.com/knative/eventing/blob/master/docs/decisions/eventing-and-messaging.md

Reply:
Hm, so is this part of the backwards compatibility? If so, could/should we have a TODO here to clarify and maybe point to an issue on why this is around and when is it going away?

Reply:
Yes, this was the old 300-channel, which will be removed in 0.9. Is the provisioner-based one.
Added a TODO and linked to the [issue](https://github.com/knative/eventing/issues/1561)